app/views/leads/index.html.erb

```diff
@@ -27,7 +27,7 @@
               </tr>
             </thead>
             <tbody>
-              <tr v-for="lead in leads">
+              <tr v-for="lead in leads" v-on:click="showEvents(lead)" v-bind:id="'lead-' + lead.id">
```

Comment:
Here's another way to approach rendering the extra row with vue in the template:
          `<tbody>
              <template v-for="lead in leads">
                <tr @click="lead.showEvents = !lead.showEvents" :key="lead.id">
                  <td>{{ moment(lead.created_at).format('dddd MMM Do YYYY, h:mm a') }}</td>
                  <td>{{ lead.first_name }}</td>
                  <td>{{ lead.last_name }}</td>
                  <td><a v-bind:href="'/leads/' + lead.id + '/edit'">{{ lead.email }}</a></td>
                  <td>{{ lead.phone }}</td>
                  <td>{{ moment(lead.appointment_date).format('dddd MMM Do YYYY, h:mm a') }}</td>
                  <td>{{ lead.notes }}</td>
                </tr>
                <tr v-bind:id="'events-' + lead.id" v-if="lead.showEvents" style="background-color:lightgrey;">
                  <td colspan="7">
                  <div v-for="event in lead.events">
                    <div class="row">
                      <div class="col-md-2" style="margin-bottom:10px;">
                        {{ event.name }}
                      </div>
                      <div class="col-md-4" style="margin-bottom:10px;">
                        {{ moment(event.created_at).format('dddd MMM Do YYYY, h:mm a') }}
                      </div>
                    </div>
                  </td>
                </tr>
              </template>
            </tbody>`

And in the javascript just this:
`mounted: function() {
      $.get('/api/v1/leads.json').success(function(response) {
        this.leads = response;
        this.leads.map(function(lead){
          app.$set(lead, 'showEvents', false);
        })
      }.bind(this));
    }`